pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -24,7 +24,7 @@ const (
 	httpsPrefix = "https://"
 )
 
-func getDefaultInstrumentation(agentConfig *adapters.CwaConfig) (*v1alpha1.Instrumentation, error) {
+func getDefaultInstrumentation(agentConfig *adapters.CwaConfig, extraJavaEnvVar map[string]string) (*v1alpha1.Instrumentation, error) {
```

Comment:
Does not need to be a language specific variable.

Reply:
Yes since. we have python and java. Why would these env var be in python.

Reply:
We're passing it into each `getInstrumentationInstance`, which isn't language specific. If I wanted to set extra environment variables for python and go, would I need to add another map to this?

```go
getDefaultInstrumentation(agentConfig, nil, nil, map[string]string{"go-key": "go-value"})
```

Why not just use the same map? Each getInstrumentationInstance only gets the language specific instrumentation anyways so the other languages are ignored.

Or if you want to make the map language specific, make it a map of maps.

```
map[Type]map[string]string
```

The `annotation_type.go` file has a defined enum for java/python.

Reply:
I like map[Type]map[string]string